src/main/java/gregtech/common/pipelike/itempipe/longdistance/MetaTileEntityLDItemEndpoint.java

```diff
@@ -0,0 +1,54 @@
+package gregtech.common.pipelike.itempipe.longdistance;
+
+import codechicken.lib.render.CCRenderState;
+import codechicken.lib.render.pipeline.ColourMultiplier;
+import codechicken.lib.render.pipeline.IVertexOperation;
+import codechicken.lib.vec.Matrix4;
+import gregtech.api.GTValues;
+import gregtech.api.metatileentity.MetaTileEntity;
+import gregtech.api.metatileentity.interfaces.IGregTechTileEntity;
+import gregtech.api.pipenet.longdist.MetaTileEntityLongDistanceEndpoint;
+import gregtech.api.util.GTUtility;
+import gregtech.client.renderer.texture.Textures;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.items.CapabilityItemHandler;
+import org.apache.commons.lang3.ArrayUtils;
+
+public class MetaTileEntityLDItemEndpoint extends MetaTileEntityLongDistanceEndpoint {
```

Comment:
This class should override `getIsWeatherOrTerrainResistant()` to return true, and additionally add the `"gregtech.universal.tooltip.terrain_resist"` tooltip. See `MetaTileEntityFisher` for example implementation details.